src/attestUtxo.c

```diff
@@ -293,38 +291,38 @@ enum {
 void attestUtxo_sendResponse(attestUtxoState_t* state)
 {
 	// Response is (txHash, outputNumber, outputAmount, HMAC)
-	uint8_t response[32 + 4 + 8 + 16]; // Note: we have short HMAC
-	uint8_t hmac[32]; // Full HMAC
+	uint8_t responseBuffer[32 + 4 + 8 + 16]; // Note: we have short HMAC
+	uint8_t hmacBuffer[32]; // Full HMAC
 
 	size_t pos = 0;
 
 	// txHash
-	blake2b_256_finalize(&state ->txHashCtx, response + pos, 32);
+	blake2b_256_finalize(&state ->txHashCtx, responseBuffer + pos, 32);
```

Comment:
Note -- maybe instead of `pos` we should have `responseWritePtr` a-la the way I have done it in keyDerivation

Reply:
The comment wasn't about introducing macro's per se -- it was about changing "responseBuffer + pos" into pointer